jobs/e2e_runner.groovy

```diff
@@ -98,7 +98,8 @@ evaluate(new File("${workspace}/common.groovy"))
         make docker-test
 
         docker login -e="\$DOCKER_EMAIL" -u="\$DOCKER_USERNAME" -p="\$DOCKER_PASSWORD"
-        DEIS_REGISTRY='' make docker-build ${dockerPush}
+        # build once with "docker --pull --no-cache" to avoid stale layers
```

Comment:
This won't have to be changed if/when https://github.com/deis/jenkins-jobs/pull/325 is merged; though no biggie to resolve merge conflict depending on order...